feature_engine/timeseries/forecasting/expanding_window_features.py

```diff
@@ -93,6 +93,9 @@ class ExpandingWindowFeatures(BaseForecastTransformer):
 
     {drop_original}
 
+    group_by: str, str, int, or list of strings or integers, default=None
```

Comment:
if we are repeating the same string over and over, instead of writing it multiple times, we'd create a single text in the `_docstrings` module, and import it instead. Like we do with `fit_transform` for example.